public/js/gogs.js

```diff
@@ -830,6 +830,10 @@ $(document).ready(function () {
 
     // Highlight JS
     if (typeof hljs != 'undefined') {
+        // creating aliases for languages
```

Comment:
> Why this is needed?

Highlight.js tries to detect the language automatically. But in this particular case, it didn't work as expected. So, I'm explicity setting the language to Highlight.js, as a `language-{{language-extension}}` class. Unfortunally, not all languages are being reconized by its file extension ([see](http://highlightjs.readthedocs.org/en/latest/css-classes-reference.html#language-names-and-aliases)). So I manually created aliases to two cases I found, but for most languages it isn't necessary.